### ISSUE TITLE
Missing  DODAG configuration

### ISSUE THREAD

**Issue by smlng** [mcr/unstrung#23]
Unstrung does not send a DODAG configuration option, as described in [RFC 6550 6.7.6](https://tools.ietf.org/html/rfc6550#section-6.7.6).

Though, I also thought its optional on first read, it finally states 

> This option ... MUST be included in response to a unicast request, e.g. a
   unicast DODAG Information Solicitation (DIS) message.

This issues popped up when I tried to run `sunshine` RPL with some sensor nodes running [RIOT](https://github.com/RIOT-OS/RIOT). RIOT is expecting this option and by default will discard any DODAG without it, as a workaround i compiled RIOT with `CFLAGS += -DGNRC_RPL_DODAG_CONF_OPTIONAL_ON_JOIN`.

**Comment by mcr** [comment]
Just to ack your report.

**Comment by mcr** [comment]
@sming, if "near future" you mean before end of June. No. By end of September, definitely.

**Comment by smlng** [comment]
No, June wasn't my time frame either ... September sounds good! If you need help, e.g., interop tests with the RPL implementation in RIOT-OS, I'm in.
